src/richie/apps/search/viewsets/courses.py

```diff
@@ -14,16 +14,16 @@
 from ..defaults import FILTERS_HARDCODED, RESOURCE_FACETS
 from ..exceptions import QueryFormatException
 
+# Get the courses indexer from the settings
+COURSE_INDEXER = import_string(settings.ES_INDICES.courses)
+
 
 class CoursesViewSet(ViewSet):
     """
     A simple viewset with GET endpoints to fetch courses
     See API Blueprint for details on consumer use
     """
 
-    # Get the courses indexer from the settings
-    indexer = import_string(settings.ES_INDICES.courses)
```

Comment:
The only reason I did it that way was an atavistic dislike for globals but I can see how this is not any better.

LGTM 👍